tools/QuietBackgroundProcesses/DevHome.QuietBackgroundProcesses.ElevatedServer/PerformanceRecorderEngineWinrt.cpp

```diff
@@ -0,0 +1,291 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+#include <pch.h>
```

Comment:
I'm confused.  Why are you using WRL?  The .h and .cpp should have been generated for you by cppwinrt.

Reply:
Using cppwinrt types don't decrement their refcount on the serverlifetime when used out-of-proc, so I rewrote in WRL :S